server/sonar-server/src/main/java/org/sonar/server/exceptions/NotFoundException.java

```diff
@@ -19,11 +19,12 @@
  */
 package org.sonar.server.exceptions;
 
+import static java.lang.String.format;
 import static java.net.HttpURLConnection.HTTP_NOT_FOUND;
 
 public class NotFoundException extends ServerException {
 
-  public NotFoundException(String message) {
-    super(HTTP_NOT_FOUND, message);
+  public NotFoundException(String message, Object... messageArguments) {
```

Comment:
Why ? Because as you know, if the message contains '%' something without arguments, the WS will badly fail....
If you want to keep it like this, you should forbid having no args => but It makes no sense in my opinion.